paasta_tools/cli/schemas/tron_schema.json

```diff
@@ -207,6 +207,20 @@
                         "type": "string"
                     }
                 },
+                "crypto_keys": {
+                    "encrypt": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "decrypt": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    }
+                },
```

Comment:
just fyi: we may need to update tron/task_processing to support crypto_keys

Reply:
where would that need to happen?

Reply:
for the payload that tron gets: probably somewhere around https://github.com/Yelp/paasta/blob/master/paasta_tools/tron_tools.py#L884

for processing in tron: probably something like https://github.com/Yelp/Tron/pull/822

and for task_processing: you'll likely want to pick up the work in https://github.com/Yelp/task_processing/pull/188/files


that said: if we're not expecting tronjobs to need this, it'll be a lot easier to just exclude this from the schema :p

Reply:
If the need arises, we can support direct authentication via AWS. So we don't need to make changes for tronjobs currently.